test/common/README.md

```diff
@@ -761,6 +762,11 @@ a full `setImmediate()` invocation passes.
 should not be in scope when `listener.ongc()` is created.
 
 
+## tick Module
+
+The `tick` module provides a helper function that can be used to call a callback
+after a given number of eevnt loop "ticks".
```

Comment:
Nit: eevnt -> event :)

Reply:
Nice catch @vsemozhetbyt! I believe I need to add a fix commit? Thx!

Reply:
It seems this is already done)